definitions/region/common.yaml

```
@@ -154,13 +155,13 @@
   - China+ (R10):
       description: Centrally-planned Asia, primarily China
       ar6: R10CHINA+
-      countries: China, Cambodia, Hong Kong, Laos, Macao, Mongolia, Myanmar,
-        North Korea, Viet Nam
+      countries: China, Hong Kong,  Macao, Mongolia, 
+        North Korea, South Korea
```

Please follow black style with a maximum line width of 88 characters
```suggestion
      countries: China, Hong Kong, Macao, Mongolia, North Korea, South Korea
```